src/libs/ReportUtils.js

```diff
@@ -1565,6 +1565,28 @@ function getProperSchemaForModifiedExpenseMessage(newValue, oldValue, valueName,
     return Localize.translateLocal('iou.updatedTheRequest', {valueName: displayValueName, newValueToDisplay, oldValueToDisplay});
 }
 
+/**
+ * Get the proper message schema for modified expense message.
+ *
+ * @param {String} newValue
+ * @param {String} oldValue
+ * @param {String} valueName
+ * @param {Boolean} valueInQuotes
+ * @returns {String}
+ */
```

Comment:
Looks like this was pasted from above but you forgot to update it

Reply:
Updated!